classes/profile_table.php

```diff
@@ -62,7 +61,6 @@ public function __construct($uniqueid) {
             '1=1'
         );
         $this->define_columns(self::COLUMNS);
-        $this->column_class('responsecode', 'text-right');
```

Comment:
Why not keep this column on its own? It's visually hard up against it so it's not much different real estate wise, except it is right aligned

Reply:
![image](https://user-images.githubusercontent.com/9924643/146694958-7643e22e-065e-4527-9171-ace08dce6448.png)

Re-added the column to the list page